app/Domain/Slack/Handlers/DailySummary.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Toby\Domain\Slack\Handlers;
+
+use Illuminate\Support\Carbon;
+use Spatie\SlashCommand\Attachment;
+use Spatie\SlashCommand\Request;
+use Spatie\SlashCommand\Response;
+use Toby\Domain\DailySummaryRetriever;
+use Toby\Eloquent\Models\User;
+use Toby\Eloquent\Models\Vacation;
+
+class DailySummary extends SignatureHandler
+{
+    protected $signature = "toby dzisiaj";
+    protected $description = "Codzienne podsumowanie";
+
+    public function handle(Request $request): Response
+    {
+        $dailySummaryRetriever = app()->make(DailySummaryRetriever::class);
+
+        $now = Carbon::today();
+
+        $absences = $dailySummaryRetriever->getAbsences($now)
+            ->map(fn(Vacation $vacation) => $vacation->user->profile->full_name);
+
+        $remoteDays = $dailySummaryRetriever->getRemoteDays($now)
+            ->map(fn(Vacation $vacation) => $vacation->user->profile->full_name);
+
+        $birthdays = $dailySummaryRetriever->getBirthdays($now)
+            ->map(fn(User $user) => $user->profile->full_name);
+
+        $absencesAttachment = Attachment::create()
+            ->setTitle("Nieobecności :sunny:")
+            ->setColor("#eab308")
+            ->setText($absences->isNotEmpty() ? $absences->implode("\n") : "Wszyscy dzisiaj pracują :muscle:");
+
+        $remoteAttachment = Attachment::create()
+            ->setTitle("Praca zdalna :house_with_garden:")
+            ->setColor("#527aba")
+            ->setText($remoteDays->isNotEmpty() ? $remoteDays->implode("\n") : "Wszyscy dzisiaj są w biurze :boom:");
+
+        $birthdayAttachment = Attachment::create()
```

Comment:
What if we would have five more activities? Maybe this could be encapsulated somehow?